.github/workflows/release.yml

```diff
@@ -97,3 +97,14 @@ jobs:
           workflow: Docker Hub Release
           repo: twilio/twilio-cli
           token: ${{ secrets.REPO_ACCESS_TOKEN }}
+  platform-executables-release:
+    runs-on: ubuntu-latest
+    needs: [ release ]
+    steps:
+      - name: Invoke Docker workflow
```

Comment:
Seems like a copy error -`Docker` ? Should change to "Invoke Platform Executables Workflow" ?